Gaming (#518)
solana-developers/solana-cookbook: docs/gaming/monetization.md

---

> title: Monetization in crypto games

Reviewer: Could we do this one in a separate PR? It's difficult to review among everything else.

head:
  - - meta
    - name: title

>       content: Solana Cookbook | Monetization in crypto games

Reviewer: ```suggestion
      content: Solana Cookbook | Monetizing your blockchain game on Solana
```
SEO hacks

  - - meta
    - name: og:title

>       content: Solana Cookbook | Monetization in crypto games

Reviewer: ```suggestion
      content: Solana Cookbook | Monetizing your blockchain game on Solana
```

  - - meta
    - name: description

>       content: Some ideas on how can you make money with your game

Reviewer: ```suggestion
      content: Learn how to monetize your blockchain game on Solana utilizing platforms, tools, and more
```

  - - meta
    - name: og:description

>       content: Some ideas on how can you make money with your game

Reviewer: ```suggestion
      content: Learn how to monetize your blockchain game on Solana utilizing platforms, tools, and more
```

  - - meta
    - name: og:image
      content: https://solanacookbook.com/cookbook-sharing-card.png
  - - meta
    - name: og:image:alt
      content: Solana splash card
  - - meta
    - name: twitter:card
      content: summary
  - - meta
    - name: twitter:site
      content: "@solanacookbook"
  - - meta
    - name: twitter:image
      content: "https://solanacookbook.com/cookbook-sharing-card.png"
  - - meta
    - name: robots
      content: index,follow,noodp
  - - meta
    - name: googlebot
      content: index,follow
footer: MIT Licensed
---

# Monetization

Building a game on Solana opens up a lot of interesting monetization options for games.

> ## Selling NFTs

Reviewer: ```suggestion
## Digital Collectibles
```

> You can sell NFTs to raise raise money. It is basically like crowd sourcing the development of your game. If the game becomes successful the value of the NFTs will likely also go up. For that you first need to create an NFT collection.

Reviewer: ```suggestion
Digital collectibles on blockchains are more commonly known as NFTs. Games can sell NFTs much like people sell items within the Steam shop. NFTs are commonly a part of a collection.
```

Removing number go up wording

[Metaplex Docs](https://docs.metaplex.com/)<br />
[Setup a candy machine step by step](https://youtu.be/0KHv1dMV8zU)<br />

> Then you can list the collection on one of the many Nft market places on Solana.

Reviewer: ```suggestion
```

## Royalty fees
NFTs on Solana can define a resell fee for NFTs which most market places read and use. This means that on every NFT sell on a secondary market place you will get a percentage of the selling price. This can be a constant income stream if the NFTs are traded a lot.
This is how it would look like if the NFT defines a 4.2% resell fee which is split between creator 1 and 2.

```js
"data":{

>     "name":string"Example NFT"

Reviewer: ```suggestion
    "name": "Example NFT"
```

>     "symbol":string"SYMBOl"

Reviewer: ```suggestion
    "symbol": "SYMBOl"
```

    "uri":string"Url to meta data"
    "sellerFeeBasisPoints":int 420
    "creators":[
        {
            "address":string"creator 1 public key"
            "verified":int 0
            "share":int 50
        },
        {
            "address":string"creator 2 public key"
            "verified":int 0
            "share":int 50
        }
        ]
    }

> "primarySaleHappened":int 1

Reviewer: Can you fix all the extra stuff that make this json invalid?
